### ISSUE TITLE
Ubuntu deb package broken with nonexec /tmp

### ISSUE THREAD

**Issue by fluential** [wazuh/wazuh#763]
Wazuh ubuntu package pre/post installation scripts heavily relay on /tmp having executable flag on.
Often this is not the case and mountpoints like /tmp are nonexectuable .
Ubuntu package pre/post deb script needs to support TMPDIR variable to be set from outside to workaround that.

 To replicate: 
```
> mount -o remount,noexec /tmp
> apt-get install wazuh-agent
The following NEW packages will be installed:
  wazuh-agent
0 upgraded, 1 newly installed, 0 to remove and 111 not upgraded.
Need to get 0 B/10.2 MB of archives.
After this operation, 108 MB of additional disk space will be used.
Preconfiguring packages ...
Can't exec "/tmp/wazuh-agent.config.RHxwOU": Permission denied at /usr/share/perl/5.18/IPC/Open3.pm line 173.
open2: exec of /tmp/wazuh-agent.config.RHxwOU configure  failed at /usr/share/perl5/Debconf/ConfModule.pm line 59.
Selecting previously unselected package wazuh-agent.
(Reading database ... 95889 files and directories currently installed.)
Preparing to unpack .../wazuh-agent_3.2.4-1_amd64.deb ...
Unpacking wazuh-agent (3.2.4-1) ...
Setting up wazuh-agent (3.2.4-1) ...
```

**Comment by BraulioV** [comment]
Hi @fluential, 

this bug will happen in the rpm and deb packages. All of them, as you said, heavily relay in the `/tmp` directory to perform some operations. So, what we are going to do is to use the `/var/ossec/tmp` directory to store files while upgrading, etc. instead of using the `/tmp` directory.

Thanks for your feedback.

**Comment by pillarsdotnet** [comment]
Any progress on this?

**Comment by BraulioV** [comment]
Hi all,

this has been solved in this PR https://github.com/wazuh/wazuh-packages/pull/66 and all the changes will be available in the Wazuh v3.7.0 packages.

I will close this issue, but if you have more questions, feel free to contact us.

Best regards,
Braulio.
